tasks/85_bet-validation/bet-validation.js

```diff
@@ -1,6 +1,7 @@
 // eslint-disable-next-line camelcase
-export const validate_bet = (arrRange, str) => {
-  let pattern = /[^0-9|\s|,]/;
+export const validate_bet = (arrRange, ...num) => {
+  const str = typeof num[0] === 'number' ? num.join(',') : num[0];
```

Comment:
We probably had some misunderstanding about sorting, I meant to update your sort function below to `.sort((a, b) => a - b)`